core/src/main/java/com/linecorp/armeria/common/DefaultFlagsProvider.java

```diff
@@ -411,4 +412,9 @@ public Sampler<? super RequestContext> requestContextLeakDetectionSampler() {
     public MeterRegistry meterRegistry() {
         return Metrics.globalRegistry;
     }
+
+    @Override
+    public Duration defaultUnloggedExceptionReportIntervalMillis() {
```

Comment:
Since we're not dealing with millisecond precision but a `Duration` object

```suggestion
    public Duration defaultUnloggedExceptionReportInterval() {
```

Reply:
Saw the discussion comments late 😅